### PR TITLE
Script to update torch and torch_xla installation

### PR DESCRIPTION
**Pull request by jysohn23**
Usage:
  for nightly update: ./scripts/update_torch_wheels.sh
  for specific build: ./scripts/update_torch_wheels.sh YYYYMMDD

### CODE REVIEW

File: scripts/update_torch_wheels.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# Must be YYYYMMDD format
+DATE=$1
+TORCH_WHEEL="gs://pytorch-tpu-releases/wheels/torch-1.1.0a0+stable-cp35-cp35m-linux_x86_64.whl"
```
Comment (dlibenzi) [inline scripts/update_torch_wheels.sh]: I'd isolate wheel file names into separate variables.

File: scripts/update_torch_wheels.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  printf 'Which of the following torch wheels would you link to install?\n'
+  gsutil ls "gs://pytorch-tpu-releases/wheels/torch-${DATE}-*"
+  printf '\n(copy paste and enter one of above): '
+  read TORCH_WHEEL
+  printf '\nWhich of the following torch_xla wheels would you link to install?\n'
+  gsutil ls "gs://pytorch-tpu-releases/wheels/torch_xla-${DATE}-*"
+  printf '\n(copy paste and enter one of above): '
+  read TORCH_XLA_WHEEL
+fi
+gsutil cp "$TORCH_WHEEL" /tmp/.
```
Comment (dlibenzi) [inline scripts/update_torch_wheels.sh]: I'd remove the dot after /tmp, and cleanup the files after install. /tmp is RAM ;)

File: scripts/update_torch_wheels.sh (hunk 1 above)
Comment (jysohn23) [reply scripts/update_torch_wheels.sh]: Comments addressed in: https://github.com/pytorch/xla/pull/681

File: scripts/update_torch_wheels.sh (hunk 2 above)
Comment (jysohn23) [reply scripts/update_torch_wheels.sh]: Thanks! Comments addressed in: https://github.com/pytorch/xla/pull/681

**Pull request closed without merging**